src/test/shell/bazel/toolchain_test.sh

```diff
@@ -533,9 +533,8 @@ EOF
     --toolchain_resolution_debug \
     --incompatible_auto_configure_host_platform \
     //demo:use &> $TEST_log || fail "Build failed"
-  expect_log 'ToolchainResolution: Looking for toolchain of type //toolchain:test_toolchain'
-  expect_log 'ToolchainResolution:   For toolchain type //toolchain:test_toolchain, possible execution platforms and toolchains: {@local_config_platform//:host -> //:test_toolchain_impl_1}'
-  expect_log 'ToolchainResolution: Selected execution platform @local_config_platform//:host, type //toolchain:test_toolchain -> toolchain //:test_toolchain_impl_1'
+  expect_log 'ToolchainResolution:   Type //toolchain:test_toolchain: target @local_config_platform//.*: execution @local_config_platform//:host: Selected toolchain //:test_toolchain_impl_1'
```

Comment:
Instead of "target @local_config_platform", which looks like it is the target being built, specify that this is "target platform @local_config_platform".

Reply:
> Does this help mitigate #7713? Also see http://b/117279695 and http://b/126375095.

#7713, b/126375095: there is still some duplications in ToolchainResolutionFunction:125, which I didn't remove because some tests depend on it (when specifying extra_toolchains)

b/117279695 is fixed with this

What would be most useful for me is to have two levels: displaying selected toolchains, displaying also why they were rejected.

Reply:
Done.